MPF.md

```diff
@@ -795,6 +795,27 @@ This example illustrates enabling the option via augments.
 
 - Introduced in 3.12.0
 
+### lastseenexpireafter
+
+This option configures the number of minutes after which last-seen entries in `cf_lastseen.lmdb` are purged.
+
+
+```json
+{
+  "variables": {
+    "default:def.control_common_ignore_missing_inputs": {
+      "value": "30240",
```

Comment:
So the specified default later

```
        string => "10080", # 10080 minutes is 1 week
```

Is also the default in the binaries I guess: https://docs.cfengine.com/docs/3.21/reference-components.html#lastseenexpireafter

Reply:
Indeed, we mirror binary defaults in the MPF as a general rule but call out when we deviate from the binary defaults.

Reply:
Noted.